features/attached_commands.feature

```diff
@@ -609,3 +541,74 @@ Feature: Command behaviour when attached to an UA subscription
            | xenial  |
            | bionic  |
            | focal   |
+
+    @series.lts
+    @uses.config.machine_type.lxd.container
+    Scenario Outline: Run timer script on an attached machine
```

Comment:
Could we just rename this test to explicitly tell we are testing the activity workflow?